src/AuthProvider_frontend/src/index.js

```diff
@@ -82,10 +82,12 @@ import { renderIndex } from './views';
 					const json = JSON.stringify(delegationChain.toJSON());
 
 					window.removeEventListener('message', listener);
+					window.location.href = `${callbackUri}?success=true&&json=` + json;
```

Comment:
Will this line redirect. bcz i have to click the button to redirect back to app on android.